tests/lib/external/prepare-key.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+set -ex
+
+INSTANCE_IP="${1:-localhost}"
+INSTANCE_PORT="${2:-8022}"
+USER="${3:-$(whoami)}"
+KEY_NAME="${4:-spread_external}"
+PASSPHRASE="${5:-}"
+
+execute_remote() {
+    # shellcheck disable=SC2029
+    ssh -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -p "$INSTANCE_PORT" "$USER@$INSTANCE_IP" "$@"
+}
+
+# Create the key in case it is not already created
+if [ ! -f "$KEY_NAME" ] || [ ! -f "$KEY_NAME".pub ]; then
```

Comment:
I would change the condition to make sure we don't accidentally nuke someone's key.
```suggestion
if [ ! -f "$KEY_NAME" ] && [ ! -f "$KEY_NAME".pub ]; then
```